### ISSUE TITLE
Request for adding localization system

### ISSUE THREAD

**Issue by qorncat** [Der-Zauberer/ServerSystemPlugin#8]
You need to add language system to reach more users. Contact me when you do. I will support.

**Comment by Der-Zauberer** [comment]
That's true, but unfortunately I don't have much time to take care of it at the moment. I'm trying to finish an update for a server, because after that I'll be on vacation and will work on other projects. I will inform you when I start working on the language system.
